src/Assets/Config.php

```diff
@@ -70,11 +70,14 @@ public static function get_relative_asset_path(): string {
 	 * @return string
 	 */
 	public static function get_url( $path ): string {
-		if ( empty( static::$path_urls[ $path ] ) ) {
-			static::$path_urls[ $path ] = trailingslashit( get_site_url() . $path );
+		$key = md5( serialize( [ WP_CONTENT_DIR, WP_CONTENT_URL ] ) );
```

Comment:
Similar here, lets cache either based on all or at least based on what can change during runtime.

I think a method, `get_cache_key` could be worth it.